Taskfile.yml

```
@@ -116,3 +116,22 @@ tasks:
       - task: npm:install-deps
     cmds:
       - npx markdownlint-cli "**/*.md"
+
+  ts:install-deps:
+    desc: Install TypeScript development dependencies
+    cmds:
+      - npm install
+
+  ts:lint:
+    desc: Lint TypeScript code
+    deps:
+      - task: ts:install-deps
+    cmds:
+      - npx eslint --ext .js,.jsx,.ts,.tsx .
+
+  ts:fix-lint:
+    desc: Fix TypeScript code linting violations
+    deps:
+      - task: ts:install-deps
```

```suggestion
  ts:lint:
    desc: Lint TypeScript code
    deps:
      - task: npm:install-deps
    cmds:
      - npx eslint --ext .js,.jsx,.ts,.tsx .

  ts:fix-lint:
    desc: Fix TypeScript code linting violations
    deps:
      - task: npm:install-deps
```

This part of the template is outdated. We are now using **npm** to manage project management tool dependencies in addition to dependencies of the TypeScript codebase, so the limited scope implied by the "ts" prefix on the task gives it a scope that is more narrow than its actual usage. This is why I switched the task name to `npm:install-deps`. That task is already present in the workflow so the `ts:install-deps` task is redundant:

https://github.com/arduino/setup-protoc/blob/11d1666fa871de56e3f058e3503006e8bc7a08c4/Taskfile.yml#L35-L39